hips/draw/simple.py

```diff
@@ -172,6 +179,53 @@ def plot_mpl_hips_tile_grid(self) -> None:
         ax.imshow(self.image, origin='lower')
 
 
+class HipsDrawResult:
+    """Container class for reporting information related with fetching / drawing of HiPS tiles.
+
+    Parameters
+    ----------
+    image: `~numpy.ndarray`
+        Container for HiPS tile data
+    geometry : `~hips.utils.WCSGeometry`
+        An object of WCSGeometry
+    tile_format : {'fits', 'jpg', 'png'}
+        Format of HiPS tile
+    """
+
+    def __init__(self, image: np.ndarray, geometry: WCSGeometry, tile_format: str) -> None:
+        self.image = image
+        self.geometry = geometry
+        self.tile_format = tile_format
+
+    def __str__(self):
+        return (
+            'This class object contains two attributes: image and geometry'
```

Comment:
This isn't useful. The idea of having a separate `__str__` is that you give more information.
In this case, it could be something like this:
```
def __str__(self):
    ss = 'HiPS draw result:\n'
    ss += 'Sky image: .... infos about sky image pixel data (shape, dtype) ...\n'
    ss += f'WCS geometry: {self.geometry}\n'
    .... later add more info about number of tiles used and fetched, time to exectute, memory used
   ... i.e. print out a nice report
    return ss
```